### DISCUSSION TITLE
Database connection Issue

### DISCUSSION THREAD

**Discussion by AbdulJabbar64** [l33t-c0d3r-66/Health-Insurance-Management-System#3] [Q&A] [answered]
I'm unable to connect to database any ideas on this issue?

**Comment by l33t-c0d3r-66** [comment, marked as answer]
Please check that your MySQL server is running
Check if it's running on port 3306
Credentials must be root for your MySQL.
If the credentials are different then place yours in this file
https://github.com/l33t-c0d3r-66/Health-Insurance-Management-System/blob/master/src/main/resources/hibernate.cfg.xml

**Reply by AbdulJabbar64** [reply]
Yes, the problem was with credentials
